source/documentation/additional-information.html.md

```diff
@@ -339,17 +348,17 @@ After submitting the above losses, customers can view their loss position. If th
 
 ## Capital Gains Tax
 
-Capital Gains Tax is a tax on the profit when someone sells or disposes of an asset that has increased in value. Customers may need to report on the following types of asset:
+Capital Gains Tax (CGT) is a tax on the profit when someone sells or disposes of an asset that has increased in value. Customers may need to report on the following types of asset:
 
 * residential property
 * other property 
 * listed and unlisted shares and securities 
 
-For more information on capital gains tax, see [Capital Gains Tax: what you pay it on, rates and allowances](https://www.gov.uk/capital-gains-tax).
+For more information on Capital Gains Tax, see [Capital Gains Tax: what you pay it on, rates and allowances](https://www.gov.uk/capital-gains-tax).
 
 MTD customers can also report any losses they have made. This charge may be set against earlier or later years. When the customer sells or disposes of the whole or part of a UK residential property and is liable to pay CGT, they must tell HMRC within a specific time following the date of completion of the sale. For more information on the reporting requirement, see [Report and pay your Capital Gains Tax: If you sold a property in the UK on or after 6 April 2020](https://www.gov.uk/report-and-pay-your-capital-gains-tax/if-you-sold-a-property-in-the-uk-on-or-after-6-april-2020). For non-resident customers, see [Tell HMRC about Capital Gains Tax on UK property or land if you’re not a UK resident](https://www.gov.uk/guidance/capital-gains-tax-for-non-residents-uk-residential-property), which advises on how these gains must be reported in year.
 
-The customer can create, update, or delete their capital gains tax in software only after the end of the tax year. If they need to make changes during the year, they need to use HMRC online services. In some cases, customers must report and pay before they submit their final declaration. However, this information will still need to be included in their final declaration if they are a Self Assessment or MTD customer.
+The customer can create, update, or delete their CGT in software only after the end of the tax year. If they need to make changes during the year, they need to use HMRC online services. In some cases, customers must report and pay before they submit their final declaration. However, this information will still need to be included in their final declaration if they are a Self Assessment or MTD customer.
 
```

Comment:
Can we write 'CGT' in normal text?

Although https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style and https://design.tax.service.gov.uk/hmrc-content-style-guide/ do not seem to outlaw 'CGT', abbreviations and acronyms are not encouraged and Capital Gains Tax is the preferred form.